ui/i18n/datepicker-ko.js

```diff
@@ -25,7 +25,7 @@ datepicker.regional.ko = {
 	dayNamesShort: [ "일","월","화","수","목","금","토" ],
 	dayNamesMin: [ "일","월","화","수","목","금","토" ],
 	weekHeader: "주",
-	dateFormat: "yy-mm-dd",
+	dateFormat: "yy. mm. dd.",
```

Comment:
Hey @scottgonzalez,

Datepicker rewrite using Globalize 1.x uses `{date: 'short'}` https://github.com/jquery/jquery-ui/blob/datepicker-globalize-1.x/ui/calendar.js#L42.

For comparison:

```js
Globalize('en').formatDate(new Date(), {date: 'short'})
// > '9/21/15'

Globalize('en').formatDate(new Date(), {date: 'medium'})
// > 'Sep 21, 2015'

Globalize('en').formatDate(new Date(), {date: 'long'})
// > 'September 21, 2015'

Globalize('en').formatDate(new Date(), {date: 'full'})
// > 'Monday, September 21, 2015'

Globalize('en').formatDate(new Date(), {skeleton: 'yMd'})
// > '9/21/2015'

Globalize('en').formatDate(new Date(2015,0,1), {skeleton: 'yMd'})
// > '1/1/2015'

Globalize('en').formatDate(new Date(2015,0,1), {skeleton: 'yMMdd'})
// > Should give you the above using padded months and days when the PR below is complete:
// https://github.com/jquery/globalize/pull/462
```